modules/visual_script/visual_script_builtin_funcs.h

```diff
@@ -94,6 +94,7 @@ class VisualScriptBuiltinFunc : public VisualScriptNode {
 		TYPE_OF,
 		TYPE_EXISTS,
 		TEXT_CHAR,
+		TEXT_ORD,
```

Comment:
Put it at the end of the enum. Currently enum values are stored as integers, and will thus break whenever something is added in the middle.